src/services/flowLogs/schema.graphql

```diff
@@ -12,7 +12,7 @@ type awsFlowLog implements awsOptionalService @key(fields: "id") {
   format: String @search(by: [hash, regexp, fulltext])
   maxAggregationInterval: Int @search
   tags: [awsRawTag]
-  vpc: [awsVpc] @hasInverse(field: flowLogs)
+  vpc: [awsVpc] @hasInverse(field: flowLog)
```

Comment:
A vpc can only have one flow log?

Reply:
Yes that's correct, in fact was TODO change: